### PR TITLE
stolostron/backlog#24960 Le rev cmts

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

**Comment by cadawson** [comment]
These are the updates from Le's review.

File: multicluster_engine/cluster_lifecycle/cluster_lifecycle_intro.adoc [hunk 1]
```diff
@@ -34,7 +33,6 @@ hibernate_created_cluster.adoc
 * xref:../cluster_lifecycle/access_cluster.adoc#accessing-your-cluster[Accessing your cluster]
```
Comment (swopebe) [inline multicluster_engine/cluster_lifecycle/cluster_lifecycle_intro.adoc:33]: please fix bold source here

File: multicluster_engine/cluster_lifecycle/ansible_config_cluster.adoc [hunk 2]
```diff
@@ -8,7 +8,6 @@
 * <<prerequisites-for-ansible-integration-cluster,Prerequisites>>
 * <<ansible-template-run-cluster-console,Configuring an AnsibleJob template to run on a cluster by using the console>>
 * <<ansible-template-create,Creating an AnsibleJob template>>
-* <<configuring-an-ansible-job-for-a-managed-cluster-labels,Configuring an AnsibleJob template to run on a managed cluster by using labels>>
 * <<ansible-status-job-cluster,Viewing the status of an Ansible job>>
```
Comment (swopebe) [inline multicluster_engine/cluster_lifecycle/ansible_config_cluster.adoc:11]: Best practice for tech writers and engineering is to not simply remove without the deprecation/removal notice, whihc I set up a few releases ago so we need to make sure we are not just ripping out content.

**Comment by swopebe** [comment]
@cadawson please see comments. The baremetal was deprecated...is it now removed? I am not sure if you had a chance to review that for cluster: https://github.com/stolostron/backlog/issues/24812

We should use such discipline/best practices for customer advocacy in our content.

File: multicluster_engine/cluster_lifecycle/cluster_lifecycle_intro.adoc (hunk 1 above)
Comment (cadawson) [reply multicluster_engine/cluster_lifecycle/cluster_lifecycle_intro.adoc:33]: Not sure why the Git preview wants to show this as bold, but it is not rendering that way in the output. Will take a look at it after GA.

File: multicluster_engine/cluster_lifecycle/ansible_config_cluster.adoc (hunk 2 above)
Comment (cadawson) [reply multicluster_engine/cluster_lifecycle/ansible_config_cluster.adoc:11]: Added to removed content.

**Comment by cadawson** [comment]
Added removed.

**Pull request merged**